apps/openmw/mwscript/cellextensions.cpp

```diff
@@ -46,18 +47,19 @@ namespace MWScript
 
                     ESM::Position pos;
                     MWBase::World *world = MWBase::Environment::get().getWorld();
+                    const MWWorld::Ptr playerPtr = world->getPlayerPtr();
 
                     if (world->findExteriorPosition(cell, pos))
                     {
-                        MWWorld::ActionTeleport("", pos, false).execute(world->getPlayerPtr());
-                        world->fixPosition(world->getPlayerPtr());
+                        MWWorld::ActionTeleport("", pos, false).execute(playerPtr);
+                        playerPtr.getClass().adjustPosition(playerPtr, true);
```

Comment:
You should use World::adjustPosition method so you don't need to add a class.hpp dependency.